/**:
  ros__parameters:
    width: 1920
    height: 1200
    exposure: 2100
    auto_exposure: false
    gain: 24
    fps: 168
    auto_white_balace: true
    rgain: 0.0
    bgain: 0.0
    ggain: 0.0
    time_offset: 45500000
    auto_exp_change: false
    max_exp: 3000
    min_exp: 200

    debug: true

    # image_process_params:
    bin_threshold: 90.0
    enemy_color: 1 # 1--blue 0--red

    # light_params:
    is_corner_correct: false

    angle_to_vertical_max: 35.0
    height_width_min_ratio: 1.3
    size_area_min_ratio: 0.4

    # armor_params:
    lights_angle_max_diff: 10.0
    lights_length_max_ratio: 1.5
    lights_y_max_ratio: 1.5
    width_height_min_ratio: 1.1
    width_height_max_ratio: 6.5
    max_angle: 35.0
    inside_thresh: 85.0
